Realm.xcodeproj/xcshareddata/xcschemes/SwiftUITestHost.xcscheme

```diff
@@ -69,6 +69,15 @@
       debugDocumentVersioning = "YES"
       debugServiceExtension = "internal"
       allowLocationSimulation = "YES">
+      <PreActions>
+         <ExecutionAction
+            ActionType = "Xcode.IDEStandardExecutionActionsCore.ExecutionActionType.ShellScriptAction">
+            <ActionContent
+               title = "Run Script"
+               scriptText = "killall Simulator&#10;defaults write com.apple.iphonesimulator ConnectHardwareKeyboard -bool false&#10;">
```

Comment:
Having the hardware keyboard option enabled will give a high chance of characters being missed when tests are writing to text fields.